runtime/doc/treesitter.txt

```diff
@@ -390,14 +390,21 @@ The following directives are built in:
             (#gsub! @_node ".*%.(.*)" "%1")
 <
     `trim!`                                          *treesitter-directive-trim!*
-        Trim blank lines from the end of the node. This will set a new
-        `metadata[capture_id].range`.
+        Trims whitespace from the node. Sets a new
+        `metadata[capture_id].range`. Trims blank lines from the end of the
```

Comment:
is it blank lines or only empty lines?
```suggestion
        `metadata[capture_id].range`. Trims empty lines from the end of the
```

Reply:
Whichever can contain (only) whitespace ;)

Reply:
Yes, so blank- I'll clarify here